test/jellyfish_web/integration/peer_socket_test.exs

```diff
@@ -30,6 +30,9 @@ defmodule JellyfishWeb.Integration.PeerSocketTest do
   end
 
   setup_all do
+    RoomService.list_rooms()
```

Comment:
On general note: since we are on a quest to always remove rooms after test, I found that there are some rooms left after running hls component test.
You can run `elixir --no-halt -S mix test test/jellyfish_web/controllers/component/hls_component_test.exs` and then check from dashboard.

I guess if we assure that no test leaves empty room we don't have to remove the rooms here.
Also, AFAIK the tests are async by default, so could this mess with some other tests?